algorithms/CSharp/src/Maths/NaisMithsRule.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
Rename the file from `NaisMithsRule.cs` to `naismith-rule.cs'

Reply:
Hello, i have been doing the correcting you have noted, however, in regards to the file rename i am a bit confused. Should it be with a "-" like you have noted above? Or an underscore? Because on the Nunit framework in regards to the path the dash will pop me an error.
Sorry if i get a bit tiring, am new to most of this stuff.

Reply:
Is with a dash (`-`) see the [contribution-guidelines](https://github.com/MakeContributions/DSA#contribution-guidelines)

Reply:
And you should add the file reference to this [file](https://github.com/simonkarofillidis/DSA/blob/add-NaisMithRule/algorithms/CSharp/README.md)

Reply:
Please delete the file `algorithms/CSharp/src/Maths/NaisMithsRule.cs`

Reply:
Done!